packages/node/src/generators/application/application.ts

```diff
@@ -0,0 +1,240 @@
+    },
+  };
+}
+
+function addProject(tree: Tree, options: NormalizedSchema) {
+  const project: ProjectConfiguration & NxJsonProjectConfiguration = {
+    root: options.appProjectRoot,
+    sourceRoot: join(options.appProjectRoot, 'src'),
+    projectType: 'application',
+    targets: {},
+    tags: options.parsedTags,
+  };
+  project.targets.build = getBuildConfig(project, options);
+  project.targets.serve = getServeConfig(options);
+
+  addProjectConfiguration(tree, options.name, project);
+
+  const workspace = readWorkspaceConfiguration(tree);
+
+  if (!workspace.defaultProject) {
+    workspace.defaultProject = options.name;
+    updateWorkspaceConfiguration(tree, workspace);
+  }
+}
+
+function addAppFiles(tree: Tree, options: NormalizedSchema) {
+  generateFiles(tree, join(__dirname, './files/app'), options.appProjectRoot, {
+    tmpl: '',
+    name: options.name,
+    root: options.appProjectRoot,
+    offset: offsetFromRoot(options.appProjectRoot),
+  });
+  if (options.js) {
+    toJS(tree);
+  }
+  if (options.pascalCaseFiles) {
+    logger.warn('NOTE: --pascalCaseFiles is a noop');
+  }
+}
+
+function addProxy(tree: Tree, options: NormalizedSchema) {
+  const projectConfig = readProjectConfiguration(tree, options.frontendProject);
+  if (projectConfig.targets && projectConfig.targets.serve) {
+    const pathToProxyFile = `${projectConfig.root}/proxy.conf.json`;
+    projectConfig.targets.serve.options.proxyConfig = pathToProxyFile;
+
+    if (!tree.exists(pathToProxyFile)) {
+      tree.write(
+        pathToProxyFile,
+        JSON.stringify(
+          {
+            '/api': {
+              target: 'http://localhost:3333',
+              secure: false,
+            },
+          },
+          null,
+          2
+        )
+      );
+    } else {
+      //add new entry to existing config
+      const proxyFileContent = tree.read(pathToProxyFile).toString();
+
+      const proxyModified = {
+        ...JSON.parse(proxyFileContent),
+        [`/${options.name}-api`]: {
+          target: 'http://localhost:3333',
+          secure: false,
+        },
+      };
+
+      tree.write(pathToProxyFile, JSON.stringify(proxyModified, null, 2));
+    }
+
+    updateProjectConfiguration(tree, options.frontendProject, projectConfig);
+  }
+}
+
+export async function applicationGenerator(tree: Tree, schema: Schema) {
+  const options = normalizeOptions(tree, schema);
+
+  const tasks: GeneratorCallback[] = [];
+  const initTask = await initGenerator(tree, {
+    ...options,
+    skipFormat: true,
+  });
+  tasks.push(initTask);
+
+  addAppFiles(tree, options);
+  addProject(tree, options);
+
+  if (options.linter !== Linter.None) {
+    const lintTask = await lintProjectGenerator(tree, {
+      linter: options.linter,
+      project: options.name,
+      tsConfigPaths: [
+        joinPathFragments(options.appProjectRoot, 'tsconfig.app.json'),
+      ],
+      eslintFilePatterns: [`${options.appProjectRoot}/**/*.ts`],
```

Comment:
This PR being merged suckedddddddddd for me and my convert tslint to eslint PR 😅 

It allowed me to spot this regression - you are not taking `js` into account with this pattern here

Reply:
^ @FrozenPandaz

Reply:
I have to tweak this code as part of fixing conflicts on my branch so it can potentially also just be fixed as part of that if you haven't already cut a release with this in?